Traceback (most recent call last):
  File "/projects/SARC/sarc/jobs/sacct.py", line 102, in __iter__
    converted = self.convert(entry)
  File "/projects/SARC/sarc/jobs/sacct.py", line 125, in convert
    if (key := alloc["type"]) not in tracked_resources:
KeyError: 'type'
There was a problem with this entry:
====================================
{'account': 'mila',
 'allocation_nodes': 4,
 'array': {'job_id': None,
           'limits': {'max': {'running': {'tasks': 0}}},
           'task': None,
           'task_id': None},
 'association': {'account': 'mila',
                 'cluster': 'raisin',
                 'partition': None,
                 'user': 'petitbonhomme'},
 'cluster': 'raisin',
 'comment': {'administrator': None, 'job': None, 'system': None},
 'constraints': 'x86_64&(48gb|80gb)',
 'derived_exit_code': {'return_code': 0, 'status': 'SUCCESS'},
 'exit_code': {'return_code': 0, 'status': 'SUCCESS'},
 'flags': ['CLEAR_SCHEDULING', 'STARTED_ON_BACKFILL'],
 'group': 'petitbonhomme',
 'het': {'job_id': 0, 'job_offset': None},
 'job_id': 1,
 'kill_request_user': None,
 'mcs': {'label': ''},
 'name': 'main.sh',
 'nodes': 'cn-c021',
 'partition': 'long',
 'priority': 7152,
 'qos': 'normal',
 'required': {'CPUs': 16, 'memory': 8192},
 'reservation': {'id': 0, 'name': 0},
 'state': {'current': 'CANCELLED', 'reason': 'Dependency'},
 'steps': [],
 'time': {'elapsed': 43200,
          'eligible': 1641003593,
          'end': 1676394060,
          'limit': 720.0,
          'start': 1676350860,
          'submission': 1676350800,
          'suspended': 0,
          'system': {'microseconds': 0, 'seconds': 0},
          'total': {'microseconds': 0, 'seconds': 0},
          'user': {'microseconds': 0, 'seconds': 0}},
 'tres': {'allocated': [{'requested': {'quossé ça fait icitte ça?': 'ché pas'}},
                        {'count': 2, 'id': 1, 'name': None, 'type': 'cpu'},
                        {'count': 10000, 'id': 2, 'name': None, 'type': 'mem'},
                        {'count': 1, 'id': 4, 'name': None, 'type': 'node'},
                        {'count': 1, 'id': 5, 'name': None, 'type': 'billing'},
                        {'count': 1, 'id': 1001, 'name': 'gpu', 'type': 'gres'},
                        {'count': 1,
                         'id': 1002,
                         'name': 'gpu:p100',
                         'type': 'gres'}],
          'requested': [{'count': 4, 'id': 1, 'name': None, 'type': 'cpu'},
                        {'count': 16384, 'id': 2, 'name': None, 'type': 'mem'},
                        {'count': 2, 'id': 4, 'name': None, 'type': 'node'},
                        {'count': 3, 'id': 5, 'name': None, 'type': 'billing'},
                        {'count': 4, 'id': 1001, 'name': 'gpu', 'type': 'gres'},
                        {'count': 4,
                         'id': 1002,
                         'name': 'gpu:p100',
                         'type': 'gres'}]},
 'user': 'petitbonhomme',
 'wckey': {'flags': [], 'wckey': ''},
 'working_directory': '/network/scratch/p/petitbonhomme/experience-demente'}
====================================
